src/components/App.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import WoolyWilly from './WoolyWilly'
+
+const App = ({
```

Comment:
you had mentioned making it easy to have multiple wooly-willies. i think here, we should pass an array of these willy specs and have the `App` render all available willies. this could be a good place for using `PropTypes` as documentation-code:
```javascript
App.propTypes = {
  willies: PropTypes.arrayOf({
    amplitudes: PropTypes.arrayOf(PropTypes.number),
    width: PropTypes.number,
    // ...etc
  })
}

Reply:
should these changes be in this branch? i didn't add any code to actually enable having more, just reorganized/added components so we could add more in theory. we'd have to change the store first